### PR TITLE
Support for pytorch 1.10.2 and deepspeed 0.5.10

### PR DESCRIPTION
**Pull request by taylormcnally**
Some trivial changes to get gpt-neox working with deepspeed 0.5.10 and pytorch 1.10.2. Removed logits param and unsupported  mask method.

### CODE REVIEW

**Comment by StellaAthena** [comment]
What have you tested with this, specifically? Multi-GPU? Multi-machine? Tensor parallelism? Pipeline parallelism?

**Comment by taylormcnally** [comment]
I should have tested all of the configurations before making this, but so far I've tested single machine for small to medium sized models. I'm adding a GPU and testing multi-gpu and will get back the with the results

**Comment by taylormcnally** [comment]
Tested with MP and PP for a multi GPU setup and it appears to be working. Though this is using a single machine. 

Screen Shot
